.env.example

```
@@ -1,3 +1,4 @@
+ANAF_CALLBACK_URL=
 ANAF_CLIENT_ID=
 ANAF_CLIENT_SECRET=
-ANAF_CALLBACK_URL=
\ No newline at end of file
+ANAF_TEST_VAT_NUMBER=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)